doc/main/reference/task_group_extensions.rst

```diff
@@ -87,6 +89,17 @@ Schedules the task object pointed by the ``h`` for execution.
       * ``h`` is not empty.
       * ``*this`` is the same ``task_group`` that ``h`` is created with.    
 
+.. cpp:function:: task_group_status run_and_wait(task_handle&& h)
+
+Equivalent to ``{run(h); return wait();}``, but guarantees that ``h`` runs on the current thread.
```

Comment:
Task priorities can postpone `h` and it might be executed by another thread.
```suggestion
Equivalent to ``run(h); return wait();``
```

Reply:
This is how templated method ``run_and_wait`` is described in the spec : https://spec.oneapi.io/versions/latest/elements/oneTBB/source/task_scheduler/task_group/task_group_cls.html 

Should it be changed as well ?

Reply:
Yes, I think the spec should be fixed.